src/screens/schedules/components/ConfirmedScheduleModalContent/index.tsx

```diff
@@ -32,6 +34,16 @@ const ConfirmedScheduleModalContent = ({
       (a) para definirem a plataforma onde será feita a ajuda.
     </Typography>
 
+    <DataContainer>
+      <Label>Assunto</Label>
+      <Typography variant="body2">{topic}</Typography>
```

Comment:
é um campo opcional:

```suggestion
      <Typography variant="body2">{topic || '-'}</Typography>
```